cms/static/sass/elements/_modules.scss

```diff
@@ -195,16 +196,17 @@
       }
     }
 
-    a {
+  button {
```

Comment:
It would be awesome if we could target a class instead of an element. Would you mind adding a class to the individual buttons (i.e., `button-add-component`) and then replace this element with that class?